src/nyc_trees/sass/partials/_globals.scss

```diff
@@ -299,3 +298,40 @@ label {
 .map-small {
   padding-top: 40%; // Makes height 40% of width
 }
+
+.map {
```

Comment:
I introduced a `partials/_map.scss` file recently, with a `map` class.  We should either move this class there or removing the `_map.scss` file.